curator/views/react/stations_today.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import $ from 'jquery'
+const date = require('../../../tools/date.js')()
+const globals = require('../../../tools/globals.js')
+const _ = require('underscore')
+
+class Stations_today extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      stations: []
+    }
+  }
+
+  componentWillMount() {
+    $.get("/stations_today", (stations) => {
+      this.setState({stations})
+    })
+  }
+
+  render() {
+    let {stations} = this.state
+    const delivery_date = date.this_delivery().format(globals.order_date_format).slice(0, -6)
```

Comment:
Are you slicing the date cause you do not want that format? in that case create another format in the globals tools and use that one instead of modifying the returned string. You can find the types of available formats on momentjs.com